_posts/2022-04-15-Partner-Highlight-Pureinsights-adds-OpenSearch-to-its-Discovery-Platform.markdown

```diff
@@ -0,0 +1,24 @@
+---
+layout: post
+title:  "Partner Highlight: Pureinsights adds OpenSearch to its Discovery Platform"
+authors:
+  - plewis
+date: 2022-04-15 08:50:01
+categories:
+  - partners
+  - technical-post
```

Comment:
Just caught this - please remove "-technical-post" as it causes issues with the Jekyll URL

Reply:
PR-775 has the required changes